back/spec/acceptance/nav_bar_items_spec.rb

```diff
@@ -37,12 +47,26 @@
         create :nav_bar_item, code: 'events'
       end
 
+      let(:codes) { json_response_body[:data].map { |d| d.dig(:attributes, :code) } }
+
       example_request 'List removed default NavBarItems' do
         expect(status).to eq 200
-        json_response = json_parse response_body
-        expect(json_response[:data].size).to be > 0
-        expect(json_response[:data].map { |d| d.dig(:attributes, :code) }).to include 'home'
-        expect(json_response[:data].map { |d| d.dig(:attributes, :code) }).not_to include 'events'
+        expect(json_response_body[:data].size).to be > 0
+        expect(codes).to include 'home'
+        expect(codes).to include 'proposals'
+        expect(codes).not_to include 'events'
+      end
+
+      context 'when NavBarItem is disabled by corresponding disabled feature' do
+        before do
+          SettingsService.new.deactivate_feature!('initiatives')
+        end
+
+        example_request 'Does not list removed default but feature-disabled NavBarItems' do
+          expect(codes).to include 'home'
+          expect(codes).not_to include 'proposals'
+          expect(codes).not_to include 'events'
```

Comment:
Where did you disable the `events_widget` feature? Or why do you test on "not including events" in the navbar?

Reply:
Currently, `events_widget` feature doesn't control `events` NavBarItem. I just copied this expect from the previous example.

Reply:
No indeed, my mistake. I was just wondering why we are testing for `events` not to be here.